docs/data-sources/dashboards.md

```diff
@@ -0,0 +1,149 @@
+  * `base` - The base of logarithmic scale charts. Omit or set to 0 for the default linear scale. Usually set to 10 for the traditional logarithmic scale.
+
+### Chart Source Queries
+
+The `source` mapping supports the following:
+
+  * `name` - Name of the source.
+  * `query` - Query expression to plot on the chart.
+  * `source_description` - A description for the purpose of this source.
+  * `disabled` - Whether the source is disabled.
+  * `scatter_plot_source` - For scatter plots, does this query source the X-axis or the Y-axis, `X`, or `Y`.
+  * `query_builder_enabled` - Whether or not this source line should have the query builder enabled.
+
+### Chart Settings
+
+The `chart_setting` mapping supports the following:
+
+  * `type` - Chart Type. `line` refers to the Line Plot, `scatter` to the Point Plot, `stacked-area` to the Stacked Area plot, `table` to the Tabular View, `scatterplot-xy` to Scatter Plot, `markdown-widget` to the Markdown display, and `sparkline` to the Single Stat view. Valid options are `line`, `scatterplot`, `stacked-area`, `stacked-column`, `table`, `scatterplot-xy`, `markdown-widget`, `sparkline`, `globe`, `nodemap`, `top-k`, `status-list`, and `histogram`.
+  * `max` - Max value of the Y-axis. Set to null or leave blank for auto.
+  * `line_type` - Plot interpolation type.  `linear` is default. Valid options are `linear`, `step-before`, `step-after`, `basis`, `cardinal`, and `monotone`.
+  * `stack_type` - Type of stacked chart (applicable only if chart type is `stacked`). `zero` (default) means stacked from y=0. `expand` means normalized from 0 to 1.  `wiggle` means minimize weighted changes. `silhouette` means to center the stream. Valid options are `zero`, `expand`, `wiggle`, `silhouette`, and `bars`.
+  * `windowing` - For the tabular view, whether to use the full time window for the query or the last X minutes. Valid options are `full` or `last`.
+  * `window_size` - Width, in minutes, of the time window to use for `last` windowing.
+  * `show_hosts` - For the tabular view, whether to display sources. Default is `true`.
+  * `show_labels` - For the tabular view, whether to display labels. Default is `true`.
+  * `show_raw_values` - For the tabular view, whether to display raw values. Default is `false`.
+  * `auto_column_tags` - This setting is deprecated.
+  * `column_tags` - This setting is deprecated.
+  * `tag_mode` - For the tabular view, which mode to use to determine which point tags to display. Valid options are `all`, `top`, or `custom`.
+  * `num_tags` - For the tabular view defines how many point tags to display.
+  * `custom_tags` - For the tabular view, a list of point tags to display when using the `custom` tag display mode.
+  * `group_by_source` - For the tabular view, whether to group multi metrics into a single row by a common source. If `false`, each source is displayed in its own row.  if `true`, multiple metrics for the same host are displayed as different columns in the same row.
```

Comment:
For the tabular view, whether to group multi metrics into a single row by a common source. If set to `false`, each source is displayed in its own row.  If set to `true`, multiple metrics for the same host are displayed as different columns in the same row.